### PR TITLE
feat(AWM): add relayer documentation

### PR DESCRIPTION
**Pull request by ashucoder9**
This PR adds the relayer documentation under the AWM section: https://github.com/ava-labs/awm-relayer/blob/main/README.md

### CODE REVIEW

File: configs/remoteContent.js [hunk 1]
```diff
@@ -90,6 +90,42 @@ ${updatedContent}`,
...
+            content,
+            "https://github.com/ava-labs/awm-relayer/blob/main/"
+          );
+
+          const newContent = insertLinesAfterFirstLine(updatedContent, newLines);
+
+          return {
+            filename: "relayer.md",
+            content: `---
+tags: [Teleporter, Cross-Subnet Communication, Cross-Chain Communication]
```
Comment (ashucoder9) [inline configs/remoteContent.js:115]:
```suggestion
tags: [Avalanche Warp Messaging, Relayer]
```

**Comment by martineckardt** [comment]
Fix Lint errors in this [PR](https://github.com/ava-labs/awm-relayer/pull/220) on awm-relayer repo

**Comment by meaghanfitzgerald** [comment]
closed by #1700

**Comment by meaghanfitzgerald** [comment]
@ashucoder9 the lint errors from earlier were because the awm doc wasn't added to `.gitignore` , don't forget this important step in the future! If you the path to the ignore retroactively, you can run `git rm --cached path/to/your/file` to remove it from the git diff.

**Pull request merged**